.github/workflows/self-zizmor.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
       - zizmor-check
     if: ${{ needs.zizmor-check.outputs.found-files == 'true' }}
 
-    uses: grafana/shared-workflows/.github/workflows/reusable-zizmor.yml@5793b5cefc1316d82c0d8842e6458a8a3d8cc847
+    uses: grafana/shared-workflows/.github/workflows/reusable-zizmor.yml@297fe73b6a757ed3d915f95343bbbb9d067d792f
```

Comment:
If this was changed to this, it wouldn't track `main` and instead would stay pinned at a tag [like we have here](https://github.com/grafana/grafana-opentelemetry-dotnet/blob/4239adf9f327c359905ec2a2acfef4269a762c21/.github/workflows/ci.yml#L149):

```
uses: grafana/shared-workflows/actions/get-vault-secrets@9f37f656e063f0ad0b0bfc38d49894b57d363936 # get-vault-secrets/v1.2.1
```

Reply:
yes that's right, but this action is not releasable yet (no tags/releases). people can decide if they want that action to be releasable so we can actually tag it and follow that pattern.